### ISSUE TITLE
Role: form vs region - ARIA acc name requirements and landmark computation per Core AAM

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/aria#2584]
I was reviewing the WCAG ARIA11 technique (https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11) in this PR https://github.com/w3c/wcag/pull/1023#discussion_r2235010072.

Currently, the ARIA spec requires an accessible name for the form role. However, with this merged PR https://github.com/w3c/aria/pull/2297, that requirement will be removed. Despite this, per the Core AAM specs https://w3c.github.io/core-aam/#role-map-form-nameless, a form without an accessible name won’t be exposed as a landmark, mirroring the behavior of region.

Given this, should we consider removing the accessible name requirement for the region role as well, to maintain consistency within the ARIA spec?

**Comment by rahimabdi** [comment]
> Currently, the ARIA spec requires an accessible name for the form role.

There may be conflicting guidance in the spec currently: for the ["form" role section](https://w3c.github.io/aria/#form), it states: "Authors SHOULD give each element with role form a brief label that describes the purpose of the form."

However, in the[ "Roles" section for Handling Authoring Errors](https://w3c.github.io/aria/#document-handling_author-errors_roles), it states for `form`/`region` roles: "In situations where an author has not specified names for these landmarks, it is considered an authoring error.".

Can authoring errors be both SHOULD or MUST? 

> Given this, should we consider removing the accessible name requirement for the region role as well, to maintain consistency within the ARIA spec?

No strong opinion although aligning form/region makes sense! It should also be WPT-testable. Related WPTs:

- [ form-roles.html](https://wpt.fyi/results/wai-aria/role/form-roles.html?label=experimental&label=master&aligned)
- [region-roles.html](https://wpt.fyi/results/wai-aria/role/region-roles.html?label=experimental&label=master&aligned)

**Comment by pkra** [comment]
> >  "In situations where an author has not specified names for these landmarks, it is considered an authoring error.".
> 
> Can authoring errors be both SHOULD or MUST? 

This looks like an oversight / should have been adjusted as part of #2297. We should fix it separately.

**Comment by giacomo-petri** [comment]
I think it's best to resolve this question first; if we decide to align role=region and role=form the entire section can be probably simplified.

**Comment by MarioBatusic** [comment]
A *region* or *form* lole that is not exported to the accessibility tree has no sense. Therefore IMO there are to possibilities:
1. Author MUST give names for these roles, and there is an author error if not, or
2. 2. The labeling is not required, but in this case the browsers should include the roles in the accessibility tree - without labels.

**Comment by stevefaulkner** [comment]
The reason why `role=form` needs an accessible name to be exposed is  because [`<form>`](https://html.spec.whatwg.org/multipage/forms.html#the-form-element) which has an implicit `role=form` being exposed in acc tree is problematic because the `<form>` element's primary purpose in HTML is programmatic not a UI landmark. The need for an acc name avoids it being announced as a landmark by screen readers when its placement in the DOM is not appropiate to its secondary function as a landmark.

**Comment by spectranaut** [comment]
Discussed during triage today: https://www.w3.org/2025/08/07-aria-minutes.html#8f8a

Marking as not needing a fix/change.
